### ISSUE TITLE
Register form fieldsets missing legend

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#6902]
An accessibility audit identified that there are two `<fieldset>`s on the registration form that are missing `<legend>`s. These are:

- Yes, I would like to be contacted with requests to review submissions to this journal. 
- Recaptcha

Both can have their legends hidden using the same technique as the legends for the privacy consent checkbox. But they need a legend which describes the checkbox.

**Comment by israelcefrin** [comment]
Google Recaptcha was a false positive, but it has been  fixed through this issue: https://github.com/pkp/pkp-lib/issues/9386
